test/extended/quota/resourcequota.go

```diff
@@ -216,6 +217,89 @@ var _ = g.Describe("[sig-api-machinery][Feature:ResourceQuota]", func() {
 			})
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
+
+		g.It("when exceed openshift.io/image-tags will ban to create new image references in the project [Skipped:Disconnected]", func() {
+			testProject := oc.SetupProject()
```

Comment:
you don't need to setup a new project, just `oc.Namespace()` will give you the one you need.

Reply:
Updated